bin/node/src/chain_spec.rs

```diff
@@ -109,6 +112,33 @@ impl ChainParams {
             None => "Aleph Zero Development",
         }
     }
+
+    pub fn account_ids(&self) -> Vec<AccountId> {
+        match &self.account_ids {
+            Some(ids) => ids
+                .iter()
+                .map(|id| {
+                    let id = id.as_str();
+                    let key: sr25519::Public =
+                        hex::decode(id).unwrap().as_slice().try_into().unwrap();
```

Comment:
Can you call `map(|id| hex_literal::hex!(id).into())`?

Reply:
Doesn't the `hex` macro only work with string literals?

Reply:
```
error: proc macro panicked
   --> bin/node/src/chain_spec.rs:124:21
    |
124 |                     hex![id].into()
    |                     ^^^^^^^^
    |
    = help: message: expected string literals
```
:cry: